pkg/kudoctl/kudoinit/setup/setup.go

```diff
@@ -21,6 +21,9 @@ var _ kudoinit.InstallVerifier = &Installer{}
 type Installer struct {
 	options kudoinit.Options
 	steps   []kudoinit.Step
```

Comment:
this is likely where the managerStep gets it's name... but why do we see these as steps?   These are an array of initializers... the renaming to step loses that context.

Reply:
I've renamed the slice as well. I've left the `kudoinit.Step` type for now, but I agree that this should probably be renamed in another PR